test/controllers/environments_controller_test.rb

```diff
@@ -37,6 +37,7 @@ class EnvironmentsControllerTest < ActionController::TestCase
 
   test 'should update environment' do
     patch :update, params: { id: @environment, environment: { default: @environment.default, name: @environment.name, url: @environment.url, user_id: @environment.user_id } }
+    assert_not_nil assigns(:environment)
     assert_redirected_to environment_path(assigns(:environment))
```

Comment:
I don't know why, but `assigns` seems to return `nil` all the time, causing an error when the path helper is called. The additional assertion causes the test to fail without errors.